test/kafka_ex_test.exs

```diff
@@ -1,6 +1,8 @@
 defmodule KafkaExTest do
   use ExUnit.Case, async: false
 
+  doctest KafkaEx, only: [worker_spec: 2]
```

Comment:
Can we use real tests and not doctests, `server` will not be `KafkaEx.Server` going forward, it will be a more specific server to that works for the version of kafka you are running, so the test needs a dynamic value.